tests/php/Forms/GridField/GridFieldTest.php

```diff
@@ -57,6 +62,17 @@ class GridFieldTest extends SapphireTest
         ],
     ];
 
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+        $session = Controller::curr()->getRequest()->getSession();
+        $service = Injector::inst()->get(SudoModeServiceInterface::class);
+        // deactive() isn't part of the interface
```

Comment:
```suggestion
        // deactivate() isn't part of the interface
```

Reply:
Don't forget to update this when merged up to CMS 6.

Reply:
Updated